api_v3/services/ServerNodeService.php

```diff
@@ -0,0 +1,147 @@
+<?php
+/**
+ * Server Node service
+ *
+ * @service serverNode
+ * @package api
+ * @subpackage services
+ */
+class ServerNodeService extends KalturaBaseService
+{
+	public function initService($serviceId, $serviceName, $actionName)
+	{
+		parent::initService($serviceId, $serviceName, $actionName);
+		
+		$partnerId = $this->getPartnerId();
+		if(!$this->getPartner()->getEnabledService(PermissionName::FEATURE_SERVER_NODE))
```

Comment:
If all actions of the service already protected using the API roles and permissions, why do you add that check here hard-coded?